go_modules/lib/dependabot/go_modules/update_checker.rb

```diff
@@ -11,8 +11,8 @@ module Dependabot
   module GoModules
     class UpdateChecker < Dependabot::UpdateCheckers::Base
       RESOLVABILITY_ERROR_REGEXES = [
-        # Package url/proxy doesn't include any redirect meta tags
-        /no go-import meta tags/
```

Comment:
We could still get a `no go-import meta tags` when the proxy does exist but doesn't include the required meta tags right? Should we add a spec for that as well?

Reply:
Yeah good shout, will merge this just to get main green and then follow up with a new test-case for this